tests/test-svmkit

```diff
@@ -0,0 +1,464 @@
+#!/usr/bin/env opsh
+# -*- mode: sh; sh-shell: bash -*-
+# shellcheck shell=bash disable=SC2164
+opsh::version::require v0.7.0
+
+lib::import step-runner command
+lib::import test-harness
+
+: "${PULUMI_ORG:='PULUMI_ORG-NOT-SET'}"
+: "${PULUMI_COLOR:=always}"
+: "${BRING_UP_STACK:=true}"
+: "${NODE_COUNT:=1}"
```

Comment:
NODE_COUNT for the SPEs has to be a minimum of 3 for the tests to be valid.  Otherwise it would be checking things like stake, transfers, vote account creation, or consensus.